src/index.json

```diff
@@ -25079,6 +25079,73 @@
                     "version": "2.3.1"
                 },
                 "sha256Digest": "b296b2f83a88746f8be534b61a21e8a3e51f01b57b240897f79b9f9289388643"
+            },
+            {
+                "downloadUrl": "https://azuremlsdktestpypi.blob.core.windows.net/wheels/sdk-cli-v2-public/ml-2.4.0-py3-none-any.whl",
+                "filename": "ml-2.4.0-py3-none-any.whl",
+                "metadata": {
+                    "azext.minCliCoreVersion": "2.15.0",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.6",
+                        "Programming Language :: Python :: 3.7",
+                        "Programming Language :: Python :: 3.8",
+                        "Programming Language :: Python :: 3.9",
+                        "Programming Language :: Python :: 3.10",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "azpycli@microsoft.com",
```

Comment:
Could you please update to the correct contact email?

Reply:
We were always having the same email in previous ml extension. I will update the email in next ML extension.